pkg/ipamclaimswebhook/podmutator.go

```diff
@@ -190,6 +194,26 @@ func (a *IPAMClaimsValet) Handle(ctx context.Context, request admission.Request)
 	return admission.Allowed("carry on")
 }
 
+func validateDefaultMultusNetworkRequest(pod *corev1.Pod, primaryUDNInterfaceName string) error {
+	annotationValue, exists := pod.Annotations[config.MultusDefaultNetAnnotation]
+	if !exists {
+		return nil
+	}
+
+	defaultNet, err := udn.GetK8sPodDefaultNetworkSelection(annotationValue, pod.Namespace)
+	if err != nil {
+		return fmt.Errorf("failed to parse default network annotation: %v", err)
+	}
+
+	if defaultNet != nil && defaultNet.Name != primaryUDNInterfaceName {
+		return fmt.Errorf(
+			"multus default network is only allowed on the primary UDN interface (%q), but was requested on interface %q",
```

Comment:
```suggestion
			"multus default network is only allowed on the primary UDN interface %q, but was requested on interface %q",
```

no point having the parenthesis now that you've quoted the variable.